tests/packages/test_package.py

```diff
@@ -14,3 +14,13 @@ def test_package_authors():
     package.authors.insert(0, "John Doe")
     assert package.author_name == "John Doe"
     assert package.author_email is None
+
+
+def test_author_email_only():
+    """
+    Checks Package for correct handling of unparsable authors
+    (email only for example)
+    """
+    package = Package("foo", "0.1.0")
+    package.authors.append("<support@example.com>")
+    assert not package.author_name
```

Comment:
May you add `assert package.author_email` accordingly?

Reply:
Thanks for fast reviewing. I added `assert not package.author_email`, because parsing only email would need changing `AUTHOR_REGEX`. 
Of course, I could add something like `author, email = match.author, match.email if match else None, raw_author`, but then there could be situations when someone specified only name instead of email, so idk =/

Reply:
Well... it worth changing the regex for this fix.

Reply:
Done! I had do add additional regex pattern, because it's hard to parse `support@example.com` and `<support@example.com>` in one. But that new pattern is easy to understand.